### DISCUSSION TITLE
Missing Member Browser Option In List On Left

### DISCUSSION THREAD

**Discussion by sprdave** [codefori/vscode-ibmi#606] [IBM i] [answered]
I am trying to "Get Started"  using Visual Studio Code with PUB400.com and am having issue after issue trying to make anything work.    Perhaps I am too "green" at both of them... but I will keep trying.  My issue today is I am trying to follow along with this tutorial https://www.youtube.com/watch?v=IRkCLZDv01Q&ab_channel=Yusy4Code and "I DO NOT have "Member Browser" as an option.   I have:
Help and Support
User Library List
Object Browser
IFS Browser
Database Browser

Please help me understand why I am not seeing Member Browser as list option and how to add it.  Thanks in advance.   

-Dave

**Comment by worksofliam** [comment, marked as answer]
Hi Dave - I have replied and closed your issue.

For the future, please do not make an issue and a discussion post. A discussion post is plenty for questions.

Thanks!

**Reply by sprdave** [reply]
I thought my "issue" was a "question".  Sorry if I "asked" wrong.  New here. -)  Thanks for your quick response!!
